### PR TITLE
OKTA-523483: refactor i18n transformer part2

### PR DESCRIPTION
**Pull request by shuowu-okta**
## PR Checklist

- [ ] Have you verified the basic functionality for this change?
- [ ] Did you add tests, as appropriate, following our [Automated Test guidelines](https://oktawiki.atlassian.net/wiki/spaces/eng/pages/2676497890/Automated+Testing+in+the+Signin+Widget)?
- [ ] Did you follow our [Security Best Practices](https://oktawiki.atlassian.net/wiki/display/eng/Security+Best+practices)?
- [ ] Did you verify the change by running [downstream monolith artifact](https://oktawiki.atlassian.net/wiki/spaces/eng/pages/102897979/Sign-in+Widget+Development#Sign-inWidgetDevelopment-Instructionstocreateandrunthedownstreamartifact(d16t))? (Provide link to build below)
- [ ] Does this PR include noticeable changes to the UI? (If yes, attach screenshots/video below)

### Issue:

- [OKTA-523483](https://oktainc.atlassian.net/browse/OKTA-523483)

### CODE REVIEW

File: src/v3/src/components/Widget/index.tsx [hunk 1]
```diff
@@ -76,8 +76,6 @@ export const Widget: FunctionComponent<WidgetProps> = (widgetProps) => {
   } = widgetProps;
 
   const [data, setData] = useState<FormBag['data']>({});
-  // This data is not included when submitting to auth-js
-  const [additionalData, setAdditionalData] = useState<FormBag['data']>({});
```
Comment (shuowu-okta) [inline src/v3/src/components/Widget/index.tsx:80]: @glenfannin-okta seems like authjs can do the filter work (forgot when it's added), so I removed this global context here.

File: src/v2/ion/i18nTransformer.js [hunk 2]
```diff
@@ -89,7 +89,6 @@ const I18N_OVERRIDE_MAPPINGS = {
   'select-authenticator-authenticate.authenticator.webauthn': 'oie.webauthn.label',
   'select-authenticator-authenticate.authenticator.yubikey_token': 'oie.yubikey.label',
   'select-authenticator-authenticate.authenticator.custom_app': 'oie.custom.app.authenticator.title',
-  'select-authenticator-authenticate.authenticator.custom_app.push': 'oie.custom.app.authenticator.title',
```
Comment (shuowu-okta) [inline src/v2/ion/i18nTransformer.js:92]: added in https://github.com/okta/okta-signin-widget/pull/2704

**Review by glenfannin-okta — APPROVED** [review]

File: src/v3/src/components/QRCode/QRCode.tsx [hunk 3]
```diff
@@ -14,23 +14,26 @@ import { Box } from '@mui/material';
 import { withTheme } from '@okta/odyssey-react-theme';
 import { h } from 'preact';
 import { QRCodeElement, UISchemaElementComponent } from 'src/types';
+import { getTranslation } from 'src/util';
```
Comment (glenfannin-okta) [inline src/v3/src/components/QRCode/QRCode.tsx:17]: Couldnt run locally because of this, had to change to `'../../util'`

Comment (shuowu-okta) [reply src/v3/src/components/QRCode/QRCode.tsx:17]: bacon also caught this issue, going to fix it now

**Pull request merged**